usr/src/uts/common/brand/lx/procfs/lx_prvnops.c

```diff
@@ -4995,6 +4993,24 @@ lxpr_read_sys_kernel_rand_entavl(lxpr_node_t *lxpnp, lxpr_uiobuf_t *uiobuf)
 	lxpr_uiobuf_printf(uiobuf, "%d\n", swrand_stats.ss_entEst);
 }
 
+static void
+lxpr_read_sys_kernel_rand_uuid(lxpr_node_t *lxpnp, lxpr_uiobuf_t *uiobuf)
+{
+	/*
+	 * Each read from this read-only file should return a new
+	 * random 128-bit UUID string in the standard UUID format.
+	 */
+	zone_t *zone = LXPTOZ(lxpnp);
+	char uuid[LX_BOOTID_LEN];
```

Comment:
I have given this a +1 
But one nit if you feel like addressing is we could probably pull in `sys/uuid.h` and use `UUID_PRINTABLE_STRING_LENGTH`  from there instead of `LX_BOOTID_LEN`

Reply:
Sure, can fix that. Busy now but will try to find some time before the end of the day here.

Reply:
Removed LX_BOOTID_LEN and using UUID_PRINTABLE_STRING_LENGTH instead now.